============================= test session starts ==============================
platform linux -- Python 3.11.9, pytest-8.1.1, pluggy-1.4.0
rootdir: /home/runner/work/Assessment-Backend-Week-1/Assessment-Backend-Week-1
collected 0 items / 1 error

!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.17s ===============================
************* Module app
days_api/app.py:12:0: E0401: Unable to import 'pysnooper' (import-error)
days_api/app.py:45:4: W0702: No exception type(s) specified (bare-except)
days_api/app.py:61:4: W0702: No exception type(s) specified (bare-except)
days_api/app.py:104:4: W0702: No exception type(s) specified (bare-except)
days_api/app.py:7:0: W0611: Unused Response imported from flask (unused-import)
days_api/app.py:12:0: W0611: Unused import pysnooper (unused-import)

------------------------------------------------------------------
Your code has been rated at 8.67/10 (previous run: 8.67/10, +0.00)

